Dockerfile

```diff
@@ -14,41 +17,43 @@ RUN set -eux; \
         musl-dev \
         pcre-dev \
         wget \
+    # Download and set up Coreruleset
     && wget -qO/tmp/coreruleset.tar.gz https://github.com/coreruleset/coreruleset/archive/${CORERULESET_VERSION}.tar.gz \
     && echo "$CORERULESET_MD5  /tmp/coreruleset.tar.gz" | md5sum -c \
     && mkdir -p /tmp/coraza-coreruleset \
     && mkdir -p /etc/coraza-spoa/rules \
     && tar xzf /tmp/coreruleset.tar.gz --strip-components=1 -C /tmp/coraza-coreruleset \
-    && rm /tmp/coreruleset.tar.gz \
-    && cp /tmp/coraza-coreruleset/crs-setup.conf.example /etc/coraza-spoa/crs-setup.conf \
-    && cp /tmp/coraza-coreruleset/rules/* /etc/coraza-spoa/rules/ \
-    && rm -rf /tmp/coraza-coreruleset \
-    && find /etc/coraza-spoa/rules -type f -name '*.example' | while read -r f; do cp -p "$f" "${f%.example}"; done \
-    && sed -i.example 's/^\(SecDefaultAction "phase:[12]\),log,auditlog,pass"/\1,log,noauditlog,deny,status:403"/' /etc/coraza-spoa/crs-setup.conf
+    && mv /tmp/coraza-coreruleset/crs-setup.conf.example /etc/coraza-spoa/crs-setup.conf \
+    && mv /tmp/coraza-coreruleset/rules /etc/coraza-spoa \
+    && rm -rf /tmp/*
 
+# Copy project files into build image
 COPY . .
```

Comment:
Please do check https://github.com/corazawaf/coraza-spoa/pull/34#discussion_r970558014

Reply:
I have fixed this in the latest Dockerfile commit.